docs/UserGuide.md

```diff
@@ -337,16 +337,16 @@ c=30 d=Lunch n=John w=1 n=Mary w=1
 
 ### __v1.1__
 
-### Adding a person: `add`
+### Adding a person: `addPerson`
 
 Adds a person to the address book.
 
-Format: `add n=NAME p=PHONE_NUMBER e=EMAIL a=ADDRESS [t=TAG]…​`
+Format: `addPerson n=NAME p=PHONE_NUMBER e=EMAIL a=ADDRESS [t=TAG]…​`
 
 Examples:
 
-* `add n=John Doe p=98765432 e=johnd@example.com a=John street, block 123, #01-01`
-* `add n=Betsy Crowe t=friend e=betsycrowe@example.com a=Newgate Prison p=1234567 t=criminal`
+* `addPerson  n=John Doe p=98765432 e=johnd@example.com a=John street, block 123, #01-01`
+* `addPerson  n=Betsy Crowe t=friend e=betsycrowe@example.com a=Newgate Prison p=1234567 t=criminal`
```

Comment:
Could we have a more realistic address HAHA

Reply:
Updated LOL